DEVELOPMENT.md

```diff
@@ -89,32 +82,23 @@ styled(Component)`
 `;
 ```
 
-Then with arrow-function to:
-
-```jsx
-styled(props => <Component {...props} />)`
-  ${props => baseStyles(props)}
-`;
-```
-
 ## Styling
 
-`src/components` will provide 'fi-' -prefixed classNames to all components - and then at `src/core` styling is based on those base classNames.
+components will have ’fi-’ -prefixed class names and styling is based on those class names.
```

Comment:
Maybe change to different backticks or something as now it is not looking correct:

Now it looks like this:
```
’fi-’
```

Reply:
Fixed 👍